build/bin/remove_unneeded

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Sanity check: the file is created by the Dockerfile.
+if [ ! -f /in-the-container ]; then
+  echo "Don't run this outside the container!"
+  exit 1
+fi
+
+# Array of basic tools needed by entrypoint and user_setup.
+basicNeed=("bin/sh" "/bin/bash" "/usr/bin/coreutils")
```

Comment:
I'd love to get get rid of some of these (`bash` in particular), but I think the scope is fine for a quicker `release-v1.2` fix.

Reply:
bash unfortunately is not that easy. The script is bash not sh (easier to write). I will keep this in mind for master though.